Makefile

```diff
@@ -48,9 +48,14 @@ DOCKER_TOX_TEST := sh -uexc ' \
 	&& tox \
 '
 .PHONY: build
-build:
+build: VERSION.h
 	$(CC) $(CFLAGS) -o dumb-init dumb-init.c
 
+VERSION.h: VERSION
+	echo '// THIS FILE IS AUTOMATICALLY GENERATED' > VERSION.h
+	echo '// Run `make VERSION.h` to update it after modifying VERSION.' >> VERSION.h
+	xxd -i VERSION >> VERSION.h
```

Comment:
can't really decide if this is simpler than just duplicating the version in the C file or not...

unfortunately there doesn't seem to be a good way to have the C preprocessor read from a file into a string